src/components/tabController/TabPage.tsx

```diff
@@ -68,7 +72,7 @@ export default class TabPage extends PureComponent {
     const {targetPage} = this.context;
     const {index, lazy} = this.props;
     return block([
-      cond(and(eq(targetPage, index), lazy, eq(this._loaded, 0)), [set(this._loaded, 1), call([], this.lazyLoad)]),
+      cond(and(eq(targetPage, index), Number(lazy), eq(this._loaded, 0)), [set(this._loaded, 1), call([], this.lazyLoad)]),
```

Comment:
Did you find a bug here? Does the number conversion here actually fix something?

Reply:
I think it was for typescript, but you can see it's the same on line 53.

Reply:
👍